tests/validation/tests/v3_api/test_workload.py

```diff
@@ -961,7 +958,10 @@ def test_rbac_wl_daemonset_list_negative(role, remove_resource):
         user_token = rbac_get_user_token_by_role(role)
         p_client = get_project_client_for_token(p2, user_token)
         workloads = p_client.list_workload().data
-        assert len(workloads) == 0
+        if role == CLUSTER_OWNER:
+            assert len(p_client.list_workloads(uuid=workload.uuid).data) == 0
+        else:
+            assert len(workloads) == 0
```

Comment:
@aaronRancher : For the negative test case, daemonset create step is missing. Daemonset workload should be created first and then listed for different roles.

Reply:
Addressed in changes.

Reply:
@aaronRancher : In the negative case, the daemonset should be created in the unshared project/namespace(rbac_get_unshared_project()/rbac_get_unshared_ns())
Also, can you please add the check for cluster owner role ? Only other roles have been addressed.

Reply:
Addressed in changes.